em/emResidualFunctionals.cc

```diff
@@ -0,0 +1,932 @@
+/*
+ * Copyright (C) 2011 Scientific Computation Research Center
+ *
+ * This work is open source software, licensed under the terms of the
+ * BSD license as described in the LICENSE file in the top-level directory.
+ */
+#include <iostream>
```

Comment:
the only time this is used seems to be a couple of `cout` statements for debugging. They should be removed or replaced by `PCU_ALWAYS_ASSERT_VERBOSE` (I put a comment on the lines the use `cout` as well.

Reply:
Removed.